finat/point_set.py

```diff
@@ -34,15 +34,23 @@ def expression(self):
 
 
 class PointSingleton(AbstractPointSet):
-    """Just a single point."""
+    """A point set representing a single point. It therefore has no indices an
+    a `gem.Literal` expression."""
```

Comment:
Second sentence not quite clear, "an a" certainly wrong.

Reply:
Updated